docs/source/tutorials-and-examples/tutorial.rst

```diff
@@ -382,7 +382,6 @@ in MLflow saved the model as an artifact within the run.
       .. code-block:: bash
 
           curl -X POST "http://127.0.0.1:8090/predict/" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"fixed acidity\": 6.2, \"volatile acidity\": 0.66, \"citric acid\": 0.48, \"residual sugar\": 1.2, \"chlorides\": 0.029, \"free sulfur dioxide\": 29, \"total sulfur dioxide\": 75, \"density\": 0.98, \"pH\": 3.33, \"sulphates\": 0.39, \"alcohol\": 12.8}"
-
```

Comment:
Can you revert this change because it's unrelated to this PR?

Reply:
Yes, oops.  Done below.